website/blog/2020-08-31-63updates.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Windows & macOS version updates, Hermes, and more"
+author: Kiki Saintonge, Harini Kannan, Steven Moyes
+authorURL: https://twitter.com/ReactNativeMSFT
+description: "React Native for Windows is now on a 0.63 stable release with React Native core, lighting up some fun new APIs"
+is_blog: true
+published_date: August 31, 2020
+---
+
+React Native for Windows is now on a [0.63]() stable release with React Native core, lighting up some fun new APIs.
+
+<!--truncate-->
+
+Both React Native for Windows and macOS have been bumped up a version - to 0.63 for Windows and 0.62 for macOS!
+
+This means Windows is once again up-to-date with React Native core and following this new release we're lighting up a lot of new APIs and dev experiences that make developing on Windows using React Native that much better.
+
+Check out the updated Getting Started guides for [Windows](https://microsoft.github.io/react-native-windows/docs/getting-started) and [macOS](https://microsoft.github.io/react-native-windows/docs/rnm-getting-started) to try out to new features for yourself!
+
+# Developer Experience Improvements
+With the new 0.63 version we'll be taking advantage of a few really great improvements to the developer experience that were added by the React Native core team.
+
+## LogBox
+
+![logbox-windows](assets/logbox_demo.gif)
+
+LogBox is now on by default for React Native apps on Windows! This is a huge step forward in addressing some of the pain points experienced by developers when trying to debug their apps.
+
+With this new update you can now more clearly read errors in a concise, formatted, and actionable layout that's easy on the eyes and the mind - a win-win!
+
+Check out the [official blog post](https://reactnative.dev/blog/2020/07/06/version-0.63#logbox) for more information around this feature.
+
+## Autolinking
+
+![platformcolor-windows](assets/autolinking-windows.png)
+
+Sometimes it really is the little things that make a huge difference and in this case Autolinking is no exception!
+
+Now when you install new modules and build your app we will automatically hook-up all those necessary native platform-specific code pieces for you in Windows. This will save you time and (possibly) a headache, if C++ isn't your strong suite.

```

Comment:
```suggestion
Now when you install new modules and build your app we will automatically hook-up all those necessary native platform-specific code pieces for you in Windows. This will save you time and (possibly) a headache, if C++ isn't your strong suit.
```